src/snowflake/snowpark/_internal/telemetry.py

```diff
@@ -77,6 +78,9 @@ class TelemetryField(Enum):
     LARGE_QUERY_BREAKDOWN_ENABLED = "large_query_breakdown_enabled"
     # dataframe query stats
     QUERY_PLAN_HEIGHT = "query_plan_height"
+    QUERY_PLAN_NUM_SELECTS_WITH_COMPLEXITY_MERGED = (
+        "query_plan_num_selects_with_complexity_merged"
+    )
```

Comment:
should we move these to `_internal/compiler/telemetry_constants.py`?

Reply:
actually, let's move all those over, even though those telemetry are not just issued during the compilation stage, but the information is used by the compilation stage,